ci/lint.yml

```
@@ -6,10 +6,13 @@ image_resource:
   type: docker-image
   source:
     repository: golang
-    tag: latest
+    tag: 1.16.7
```

Linting should stay as `latest` (I know it's inconsistent, but there is some logic to this. The linting job shows what could be wrong according to the latest rules, the specific build though (build.yml) should be tied to a static version to preserve the reproducibility of builds. 
```suggestion
    tag: latest
```